package.json

```diff
@@ -25,7 +25,7 @@
   },
   "homepage": "https://idle.finance",
   "scripts": {
-    "test": "npx hardhat test test/*.js",
+    "test": "npx hardhat test --network hardhat",
```

Comment:
`hardhat` is the default network and `test/*.js` it's on purpose so if you need to add in an integration test in fork launch it with `yarn test-integration`

Reply:
Using the same now.